reqwest-middleware/Cargo.toml

```diff
@@ -15,7 +15,6 @@ multipart = ["reqwest/multipart"]
 json = ["reqwest/json"]
 charset = ["reqwest/charset"]
 http2 = ["reqwest/http2"]
-rustls-tls = ["reqwest/rustls-tls"]
```

Comment:
This feature was added intentionally in #160.

Some users like to only depend on `reqwest-middleware` while importing request things from `reqwest-middleware::request` (introduced in #158). The only way for them to enable these features is to enable them on `reqwest-middleware`.

We need to keep this line.